Geometry/MuonNumbering/src/CSCNumberingScheme.cc

```diff
@@ -4,7 +4,7 @@
 #include "Geometry/MuonNumbering/interface/MuonDDDConstants.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 
-//#define LOCAL_DEBUG
+#define LOCAL_DEBUG
```

Comment:
please, remove

Reply:
For GEM migration (#28796), when I commented this line there was a problem (see mail of @fabiocos 11/02/2020)

Reply:
I'm sorry, I misunderstood the comment. Now there is only #define LOCAL_DEBUG